.github/workflows/ci.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
       matrix:
         rust:
           - stable
-          - 1.46.0
```

Comment:
Was there a specific feature/crate that was necessitating 1.51?

Reply:
1.51 is required to use `resolver = "2"` in the project's `Cargo.toml`, but it can be reverted if we don't want to build `raze` itself with the new resolver. I think we should, but it can be reverted easily enough.